tests/core/full_node/test_initial_freeze.py

```diff
@@ -59,11 +60,24 @@ async def test_transaction_freeze(self, wallet_node_30_freeze):
         response = await full_node_api.send_transaction(spend)
         assert wallet_protocol.TransactionAck.from_bytes(response.data).status == MempoolInclusionStatus.FAILED
 
+        peer = full_node_server.all_connections[node_id]
         new_spend = full_node_protocol.NewTransaction(tx.spend_bundle.name(), 1, 0)
-        response = await full_node_api.new_transaction(new_spend)
-        assert response is None
```

Comment:
does this test fail without the feature?

Reply:
response is always None with these changes

Reply:
what I meant is: "does this change to the test exercise the feature you added?"

Reply:
Yup, we add a dummy connection and check if there was a `request_transaction` sent to it